styles/variables.scss

```diff
@@ -20,13 +20,13 @@ $font-weight-regular: 400;
 $font-weight-bold: 700;
 $font-weight-dark: 900;
 
-$font-size-0: 10px;
-$font-size-1: 13px;
-$font-size-2: 15px;
-$font-size-3: 20px;
-$font-size-4: 24px;
-$font-size-5: 30px;
-$font-size-6: 35px;
-$font-size-7: 40px;
-$font-size-8: 45px;
-$font-size-9: 50px;
+$font-size-0: 1.8rem;
```

Comment:
I see you added 8px to each of the originals, now it looks slightly big haha. Shall we follow this guideline instead? Feel free to change it up a little if you think the sizes are strange.

![image](https://user-images.githubusercontent.com/50544486/134760377-2fbfea93-2222-44a9-8cc6-352efc3c75dc.png)

Source: https://w3-lab.com/website-font-size-guidelines/